packages/forklift-console-plugin/src/modules/Providers/modals/EditProviderUI/EditProviderUIModal.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import { Modify, V1beta1Provider } from '@kubev2v/types';
+import { K8sModel } from '@openshift-console/dynamic-plugin-sdk/lib/api/core-api';
+
+import { EditModalProps } from '../EditModal';
+
+export type EditProviderUIModalProps = Modify<
+  EditModalProps,
+  {
+    resource: V1beta1Provider;
+    title?: string;
+    label?: string;
+    model?: K8sModel;
+    jsonPath?: string | string[];
+    content?: string;
+  }
+>;
+
+export const EditProviderUIModal: React.FC<EditProviderUIModalProps> = (props) => {
+  switch (props.resource?.spec?.type) {
+    case 'ovirt':
+    case 'openshift':
+    case 'openstack':
+    case 'vsphere':
+    default:
+      return <></>;
```

Comment:
not yet implemented, what am i missing ?

p.s.
if the modal is not yet implemented, make this PR a WIP, and implement it in a different PR, once the modal merged, mark this one as ready for review.

Reply:
ahh, I see this is an infrastructure PR, no changes in the UI,

OK :+1: